fields/default/fields_list.json

```diff
@@ -775,6 +775,17 @@
             "base.ext"
         ]
     },
+    {
+        "path": "registration_actions.items[].owner.phone_number",
+        "description": "Регистрационные действия: Предполагаемый номер телефона владельца",
+        "types": [
+            "string"
+        ],
+        "fillable_by": [
+            "base",
```

Comment:
Source named `base` really provides this field?

Reply:
Yes